### PR TITLE
feat: 🎸 update supported polymesh private versions

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

update expected spec versions to account for upgraded polymesh private chains

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/utils/constants.ts [hunk 1]
```diff
@@ -195,6 +195,3 @@ export const GLOBAL_TOKEN_URI_NAME = 'tokenUri';
 export const GLOBAL_BASE_TOKEN_URI_NAME = 'baseTokenUri';
 
 export const ASSET_ID_PREFIX = 'modlpy/pallet_asset';
-
-// TODO @prashantasdeveloper remove this when removing dual version support
-export const BTREE_SET_PORTFOLIO_ID_SPEC_VERSION = 7000004;
```
Comment (polymath-eric) [inline src/utils/constants.ts:200]: staging and testnet are already on 7000005, so it should be enough to distinguish between v6 and v7 now

**Review by prashantasdeveloper — APPROVED** [review]

**Comment by prashantasdeveloper** [comment]
Triggered from https://github.com/PolymeshAssociation/polymesh-sdk/pull/1364#issuecomment-2463908677 by [@&ZeroWidthSpace;prashantasdeveloper](https://github.com/prashantasdeveloper).

Trying to  fast forward `alpha` (436f319cc6c5eae8c565d8d090d0b8a98d2da758) to `update-private-support` (98adf0c17c59b6f7c04b70ed0ba6054e6b41767e).

Target branch (`alpha`):

```shell
commit 436f319cc6c5eae8c565d8d090d0b8a98d2da758 (HEAD -> alpha, tag: v26.2.0-alpha.2, origin/alpha)
Author: Quinn Diggity <64798147+quinndiggitypolymath@users.noreply.github.com>
Date:   Wed Nov 6 18:25:05 2024 -0800

    feat: 🎸 adds initial `Jenkinsfile`, for transitioning to jenkins artifact builds
```

Pull request (`update-private-support`):

```shell
commit 98adf0c17c59b6f7c04b70ed0ba6054e6b41767e (pull_request/update-private-support)
Author: Eric Richardson <eric@polymesh.network>
Date:   Thu Nov 7 15:40:14 2024 -0500

    feat: 🎸 update supported polymesh private versions
    
    update expected spec versions to account for upgraded polymesh private
    chains
```
Fast forwarding `alpha` (436f319cc6c5eae8c565d8d090d0b8a98d2da758) to `update-private-support` (98adf0c17c59b6f7c04b70ed0ba6054e6b41767e).
```shell
$ git push origin 98adf0c17c59b6f7c04b70ed0ba6054e6b41767e:alpha
remote: error: GH006: Protected branch update failed for refs/heads/alpha.        
remote: 
remote: - All comments must be resolved.        
To https://github.com/PolymeshAssociation/polymesh-sdk.git
 ! [remote rejected]   98adf0c17c59b6f7c04b70ed0ba6054e6b41767e -> alpha (protected branch hook declined)
error: failed to push some refs to 'https://github.com/PolymeshAssociation/polymesh-sdk.git'

**Pull request merged**